cloud/services/routetables/routetables.go

```diff
@@ -48,22 +48,26 @@ func (s *Service) Get(ctx context.Context, spec interface{}) (interface{}, error
 
 // Reconcile gets/creates/updates a route table.
 func (s *Service) Reconcile(ctx context.Context, spec interface{}) error {
+	if !s.Scope.Vnet().IsManaged(s.Scope.Name()) {
```

Comment:
This call pattern looks odd. Do we need to pass s.Scope.Name() to IsManaged? Maybe HasOwned could check the presence of the tag versus checking the actual value.

Reply:
That's a good point. I followed the aws provider on this one https://github.com/kubernetes-sigs/cluster-api-provider-aws/blob/master/pkg/cloud/services/ec2/vpc.go#L56 so I'll ask them if there's a reason for checking the value of the tag.

My guess is the name is useful in a scenario where you have more than one cluster in the same RG.